### DISCUSSION TITLE
Manual backup now button and or entity to trigger it

### DISCUSSION THREAD

**Discussion by oderbang** [lavinir/hassio-onedrive-backup#36] [Ideas]
Would it be possible to add a manual back up button.. so you do not have to wait for the scheduled backup.. maybe also have entity to trigger it

**Comment by lavinir** [comment]
I don't think I can add a button that would do this from an Addon. This is something that I can add when there will be a dedicated UI for the addon. It's on the roadmap.

Before that however, I will be adding the option to upload backups that were not created by the addon so that should at least be a workaround for this use case I imagine.

**Comment by lavinir** [comment]
On Demand backup trigger is now available through the Web UI in version 2.0 (@oderbang )

**Reply by Shaunpottie** [reply]
Not working, been trying the whole morning

**Reply by lavinir** [reply]
Could you please elaborate on the "not working"?
What happens when you click the button? How long does it take to run a backup on your system?
If a backup is already in progress in Home assistant it isn't possible to trigger another one simultaneously.
What do you see in the logs?
